package.json

```diff
@@ -34,7 +34,7 @@
     "rollup-plugin-commonjs": "9.2.1",
     "rollup-plugin-node-resolve": "4.0.1",
     "rollup-plugin-replace": "2.1.0",
-    "rollup-plugin-typescript2": "0.19.3",
+    "rollup-plugin-typescript2": "^0.21.1",
```

Comment:
Please, remove `^`, it can break a build when future versions of this package will be released, especially that according to SemVer versions starting with `0` can change their API.

Reply:
Done in https://github.com/darkreader/darkreader/pull/1341/commits/44b9ec68343ced120a97b9f10dad3f38fec1cf4b